src/components/utility/PlacePhoto.js

```diff
@@ -0,0 +1,8 @@
+export default function PlacePhoto(photo_reference) {
+  const startUrl =
+    "https://maps.googleapis.com/maps/api/place/photo?maxheight=500&photo_reference=";
```

Comment:
should maxheight value be a param?

Reply:
It might change the size of the file the API returns? 
like if you just need a thumbnail why send the full version?

Reply:
right that's what im saying - the 500 should be injected into the string. Its a code smell to have styling done through an api call.